spec/rails_helper.rb

```diff
@@ -466,12 +466,32 @@ def synchronize(seconds = nil, errors: nil)
       Capybara::Selenium::Driver.new(app, **mobile_driver_options)
     end
 
-    if ENV["ELEVATED_UPLOADS_ID"]
-      DB.exec "SELECT setval('uploads_id_seq', 10000)"
-    else
-      DB.exec "SELECT setval('uploads_id_seq', 1)"
+    migrate_column_to_bigint(AllowedPmUser, :allowed_pm_user_id)
+    migrate_column_to_bigint(Bookmark, :bookmarkable_id)
+    migrate_column_to_bigint(IgnoredUser, :ignored_user_id)
+    migrate_column_to_bigint(PostAction, :post_action_type_id)
+    migrate_column_to_bigint(Reviewable, :target_id)
+    migrate_column_to_bigint(ReviewableHistory, :reviewable_id)
+    migrate_column_to_bigint(ReviewableScore, :reviewable_id)
+    migrate_column_to_bigint(ReviewableScore, :reviewable_score_type)
+    migrate_column_to_bigint(SidebarSectionLink, :linkable_id)
+    migrate_column_to_bigint(SidebarSectionLink, :sidebar_section_id)
+    migrate_column_to_bigint(User, :last_seen_reviewable_id)
+    migrate_column_to_bigint(User, :required_fields_version)
+
+    $columns_to_migrate_to_bigint.each do |model, column|
+      if model.is_a?(String)
+        DB.exec("ALTER TABLE #{model} ALTER #{column} TYPE bigint")
+      else
+        DB.exec("ALTER TABLE #{model.table_name} ALTER #{column} TYPE bigint")
+        model.reset_column_information
+      end
     end
 
+    DB
+      .query("SELECT sequence_name FROM information_schema.sequences WHERE data_type = 'bigint'")
+      .each { |row| DB.exec "SELECT setval('#{row.sequence_name}', #{2**32})" }
```

Comment:
```suggestion
    sequence_value = 2 ** 32

    DB
      .query("SELECT sequence_name FROM information_schema.sequences WHERE data_type = 'bigint'")
      .each { |row| DB.exec "SELECT setval('#{row.sequence_name}', #{sequence_value})" }
```

Small micro-optimisation.

Reply:
To be honest, I think I prefer to leave it as it is for readability purposes. I ran a quick test on my machine and there would be a 1s difference only if we had 100M sequences to update.